### ISSUE TITLE
Disabling user gestures

### ISSUE THREAD

**Issue by Nohus** [p-lr/MapCompose#55]
Is there an intended way of displaying a static map (with no user gestures / control)? I can of course prevent the user from interacting with the Composable in a number of less or more hacky ways, but I'm wondering if there is an intended solution. There is the `disableRotation()` method, so I was looking for something like `disableScroll()`, `disableZoom()`, and `disableGestures()`.

**Comment by p-lr** [comment]
That's right, there's no intended way to disable all gestures. However, it shouldn't be hard to add.
If you have the time you can go ahead, or I'll do it myself.

**Comment by Nohus** [comment]
I'll work on a PR then.
